amanuensis/resources/userdatamodel/__init__.py

```diff
@@ -3,12 +3,16 @@
 at an administration level of the projects, cloud providers and buckets on the
 database.
 """
+from .project import *
+from .project_has_associated_user import *
+from .associated_user_roles import *
+from .associated_users import *
+from .request import *
+from .request_has_state import *
+from .state import *
+from .consortium_data_contributor import *
+from .search import *
+from .project_has_associated_user import *
+from .transition import *
+from .project_has_associated_user import *
```

Comment:
since you re-added message in the last commit you should readd it here as well otherwise the service will not find it